src/jsonrpccpp/client/connectors/redisclient.h

```diff
@@ -0,0 +1,81 @@
+/*************************************************************************
+ * libjson-rpc-cpp
+ *************************************************************************
+ * @file    redisclient.h
+ * @date    12.08.2017
+ * @author  Jacques Software <software@jacques.com.au>
+ * @license See attached LICENSE.txt
+ ************************************************************************/
+
+#ifndef JSONRPC_CPP_REDISCLIENT_H_
+#define JSONRPC_CPP_REDISCLIENT_H_
+
+#include "../iclientconnector.h"
+#include <jsonrpccpp/common/exception.h>
+#include <hiredis/hiredis.h>
+
+namespace jsonrpc
+{
+    /**
+     * This class is the redis implementation of an AbstractClientConnector.
+     * It uses hiredis to connect to the redis server to send and receive with
+     * the RedisServer connector.
```

Comment:
Please also describe which redis commands are used and how the Queues are used. 
From what I saw the server always listens on a incoming queue and then responds to a separate response queue which is random for each client, right?

This should be mentioned in the comments.